packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php

```diff
@@ -171,7 +171,7 @@
                             </div>
                         @endif
                         <div class="invoice-text">
-                            <span>{{ strtoupper(__('admin::app.sales.invoices.invoice')) }}</span>
+                            <span>{{ strtoupper(__('shop::app.customers.account.orders.view.invoices.invoice-pdf.invoice')) }}</span>
```

Comment:
Please re-arrange the '**lang key**' for all lines to match the folder structure, like this:
Like this :-  **('shop::app.customers.account.orders.invoice-pdf')**